pom.xml

```diff
@@ -67,41 +77,48 @@ limitations under the License.
   </pluginRepositories>
 
   <dependencies>
-    <dependency>
-      <groupId>com.google.guava</groupId>
-      <artifactId>guava</artifactId>
-      <version>17.0</version>
-    </dependency>
     <dependency>
        <groupId>jfree</groupId>
        <artifactId>jfreechart</artifactId>
        <version>1.0.13</version>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>
+      <version>1.9.5</version>
+    </dependency>
+    <dependency>
+      <groupId>org.mortbay.jetty</groupId>
+      <artifactId>jetty-util</artifactId>
```

Comment:
Are jetty dependencies really needed?

Reply:
@mythsunwind did you want to respond to this comment?

Reply:
Sorry, it seems i missed this question. It is only a test dependency anyway, but I removed the dependency now and made sure the tests are still running.